### ISSUE TITLE
Move to GhPages ?

### ISSUE THREAD

**Issue by nimaen** [vladotesanovic/ngSemantic#72]
Could be nice to move to Github Pages instead of Heroku, or to have both to compare 😄 
Heroku is somehow kinda slow to me...

**Comment by vladotesanovic** [comment]
I tried to deploy here:
https://vladotesanovic.github.io/ngSemantic/

But i am getting 404 errors,

What i did is very basic setup:

![image](https://cloud.githubusercontent.com/assets/738638/18229973/048b1374-728e-11e6-9849-84e933446b27.png)

**Comment by nimaen** [comment]
Yes indeed, Gh Pages is usually very simple for static websites.
Have a check here and tell me if it can help :)
http://developer.telerik.com/featured/quick-angular-2-hosting-angular-cli-github-pages/

**Comment by nimaen** [comment]
BTW, the angular 2 cli project allow you to deploy easily to gh-pages.
There might be some insights to take from there :)

**Comment by vladotesanovic** [comment]
Yep, i ll try later with angular-cli

angular-cli, component use ```template``` for displaying html while we use ```templateUrl``` 

Seems that github follow links in index.html to make them available. 

We have two solutions:

1. Create gh-pages repo
2. Serve everything from /docs folder

**Comment by nimaen** [comment]
Hum alright, I'll try to have a look by this evening.
Got a lot of stuff to work on already 😄

**Comment by vladotesanovic** [comment]
In my opinion we should not rush into this or spend to much time on this, Heroku is just fine ( maybe little slower as you said ) for now.

**Comment by nimaen** [comment]
Indeed 😄
